### ISSUE TITLE
how to propagate edge features 

### ISSUE THREAD

**Issue by may5hap** [pyg-team/pytorch_geometric#1554]
## ❓ Questions & Help

I am new to gnn and I am trying to realize some model.Now my model is described as:h_l=u(h_l,sum(m(h_l,h_k))over all k),where h_k is the feature of edge k which is the adjacent edge of edge l,m denotes the message function and u is the update function.The models I have looked have a fundamental difference from my model.so the problem is how to realize the propagation of edge features through node.thanks.

**Comment by rusty1s** [comment]
This is usually done by converting your graph into a line graph, where edges are now represented as nodes. You can do this via `torch_geometric.transforms.LineGraph`.

**Comment by may5hap** [comment]
Thanks,But while my model is being trained,LineGraph always fail to work.In the beginning of the traning,the output of LineGraph is `linegraph: Data(edge_index=[2, 72], x=[22, 8])` this is ok,but after some steps of training ,the output of LineGraph is 
`linegraph: Data(edge_index=[2, 234])` ,this is not what i want cause my input graph has 'edge_attr',the corresponding linegraph should have 'x'.So I do not know why sometimes it works but sometimes it doesn't work.
Also,I found sometimes dim of the ouput linegraph is not always the same for the graph in same topology.
Thanks for your reply.

**Comment by may5hap** [comment]
well,there may be something wrong with my data i guess.now it works well and thanks for your reply.
